util/syspolicy/policy_keys.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package syspolicy
+
+type Key string
+
+const (
+	// Keys with a string value
+	ControlURL Key = "LoginURL"  // default ""; if blank, ipn uses ipn.DefaultControlURL
+	LogTarget  Key = "LogTarget" // if blank, logging uses logtail.DefaultHost
+
+	// Keys with a string value that specifies an option: "always", "never", "user-decides"
```

Comment:
```suggestion
	// Keys with a string value that specifies an option: "always", "never", "user-decides"
	// The default is "user-decides" unless otherwise stated.
```